src/modules/recommendations.js

```diff
@@ -214,7 +216,9 @@ class Recommendations {
     const controller = new AbortController();
     const { signal } = controller;
     const headers = {};
-    const requestUrl = `${serviceUrl}/v1/recommendation_pods?key=${apiKey}`;
+    const rawQueryParams = qs.stringify(parameters);
```

Comment:
Not sure if we want to set up a guard here that ensures parameters only includes valid ones (in this case section) to prevent wrong query params from passing through. Ultimately it´s up to the consumer though, but dunno how we want to / usually handle that.